tutorials/iot/ubuntu-kiosk/electron-kiosk.md

```diff
@@ -184,7 +179,7 @@ and on your desktop, install, connect the required interface(s) and run:
 
 ```bash
 sudo snap install --dangerous ./electron-hello-world-kiosk_0.1_amd64.snap
-sudo snap connect electron-hello-world-kiosk:browser-sandbox :browser-support
+sudo snap connect electron-hello-world-kiosk:wayland
```

Comment:
`:wayland` gets autoconnected, so it can safely go away.

Reply:
`wayland` does not autoconnect (on desktop) if you have mir-kiosk installed. It is needed here.